### PR TITLE
Profile certifications

### PR DESCRIPTION
**Pull request by emiliogonmar2002**
### Components

`ProfileContainer.tsx`
- Added more data to dummy json

`ProfileCertifications.tsx`
- Receives user and certifications as props
- Renders `ProfileCertificationCard.tsx` components
- Styled

### Test Plan
image

### CODE REVIEW

File: client/src/components/ProfileCertifications/ProfileCertifications.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
```
Comment (MrFiddle) [inline client/src/components/ProfileCertifications/ProfileCertifications.tsx:1]: Same as the past PR, no need to have the React import in Vite.

**Review by MrFiddle — CHANGES_REQUESTED** [review]
Just a little detail, all good.

File: client/src/components/ProfileCertifications/ProfileCertifications.tsx (hunk 1 above)
Comment (emiliogonmar2002) [reply client/src/components/ProfileCertifications/ProfileCertifications.tsx:1]: I do need this import because I'm using <React.Fragment> component

Comment (MrFiddle) [reply client/src/components/ProfileCertifications/ProfileCertifications.tsx:1]: Oh! That's right, I forgot about that component, my bad.

**Pull request merged**